### ISSUE TITLE
Arabic language word and letter spacing do not work

### ISSUE THREAD

**Issue by ways2read** [edrlab/thorium-reader#1207]
The text adjustment for line spacing works in Arabic language content, but word and letter spacing adjustments do nothing.

**Comment by danielweck** [comment]
I suggest moving this issue to https://github.com/readium/readium-css/issues
(because Thorium uses ReadiumCSS's out-of-the-box `word-spacing` and `letter-spacing` CSS strategies, which I believe are just vanilla CSS)
Note that Thorium is built on top of the Chromium web browser engine, so perhaps CSS support for Arabic is not complete.

**Comment by danielweck** [comment]
CC @JayPanoz

**Comment by JayPanoz** [comment]
Yeah at the time we dealt with Internationalisation, so researching all the W3C docs available for different scripts, it turned out that for RTL scripts hyphenation, letter- and word-spacing weren’t a thing in terms of typography. So these are [listed as settings to disable for these scripts in Readium CSS docs](https://readium.org/readium-css/docs/CSS12-user_prefs.html#right-to-left-scripts).

Aside: if that is not implemented in some browsers maybe this would explain why. 

Given this is used for accessibility, I made sure to research that as well. There weren’t a lot of docs/studies/etc. but I found out that an equivalent would be disabling ligatures (because in Arabic for instance you have Kashida elongation). This is listed in the settings to add, although I’m not sure browsers support that very well or at all yet. 

If things have changed during these 2 years re. letter- and word-spacing though, and we have docs proving they can be useful for RTL scripts then obviously we would have to update Readium CSS.
